games/static/games/proto.js

```diff
@@ -33,22 +33,30 @@ class Square {
         return this.borders.contains('left')
     }
 
+    isRightBorder() {
+        return this.borders.contains('right')
+    }
+
+    isGround() {
+        return this.borders.contains('ground')
+    }
+
     setCharacter(character) {
         this.character = character
         let cssClass = character.getCssClass()
-        this.div.classList.clear()
+        this.div.classList.clear
```

Comment:
`clear` sounds like a function you need to activate by adding `()` , isnt it ?

Reply:
for some reason it works like this and doesnt work with ()